neighbor_test.go

```diff
@@ -0,0 +1,37 @@
+package frontman
+
+import (
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+func TestNeighbor(t *testing.T) {
+	// XXX 1. spawn one "neighbor" on another port for the test
```

Comment:
What does 'XXX' mean?
Does that comment actually documentates the code below or it's just a plan for testing the new feature?

Reply:
removed